converter/converter.py

```diff
@@ -7,21 +8,21 @@
 # Convert temperature
 
 
-def cel_to_fahren(temperature_in_celsium):
+def cel_to_fahren(temperature_in_celsium: float):
```

Comment:
@MariaMaximova, maybe you can fix the method naming?
```suggestion
def cel_to_fahrenheit(temperature_in_celsius: float):
```

Reply:
Resolved in this [commit](https://github.com/MariaMaximova/convert_temp/pull/9/commits/ea07c0df2d274ab51fb7e4346d05465b52c56248)